components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -340,9 +438,35 @@ std::vector<mojom::FeedItemV2Ptr> GenerateBlock(
   auto follow_count = GetNormal(block_min_inline, block_max_inline + 1);
   for (auto i = 0; i < follow_count; ++i) {
     bool is_discover = base::RandDouble() < inline_discovery_ratio;
-    auto generated = is_discover ? PickDiscoveryArticleAndRemove(articles)
-                                 : PickRouletteAndRemove(articles);
+    mojom::FeedItemMetadataPtr generated;
+
+    if (is_discover) {
+      generated = PickDiscoveryArticleAndRemove(articles);
```

Comment:
Is it okay that this ignores `force_content_group`?

Reply:
yes. Unless we also want to do "equitable" discovery, which could be an idea